packages/cattle-pushprox/templates/pushprox-clients.yaml

```diff
@@ -0,0 +1,159 @@
+{{- if .Values.clients }}{{- if .Values.clients.enabled }}
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "pushProxy.client.name" . }}
+  namespace: {{ template "pushprox.namespace" . }}
+  labels: {{ include "pushProxy.client.labels" . | nindent 4 }}
+    pushprox-exporter: "client"
+spec:
+  selector:
+    matchLabels: {{ include "pushProxy.client.labels" . | nindent 6 }}
+  template:
+    metadata:
+      labels: {{ include "pushProxy.client.labels" . | nindent 8 }}
+    spec: 
+{{- if .Values.clients.nodeSelector }}
+      nodeSelector: {{ toYaml .Values.clients.nodeSelector | nindent 8 }}
+{{- end }}
+{{- if .Values.clients.tolerations }}
+      tolerations: {{ toYaml .Values.clients.tolerations | nindent 6 }}
+{{- end }}
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      {{- if and .Values.clients.https.enabled .Values.clients.https.useServiceAccountCredentials }}
+      serviceAccountName: {{ template "pushProxy.client.name" . }}
+      {{- end }}
+      containers:
+      - name: pushprox-client
+        image: arvindiyengar/pushprox-linux-amd64:add_flag_for_token_path
```

Comment:
Should be a rancher hosted image.

Reply:
This is pending merges from https://github.com/rancher/PushProx/pull/1 and https://github.com/rancher/PushProx/pull/2 for initializing the Rancher repo where the hosted image will be generated from